app/controllers/posts_controller.rb

```diff
@@ -321,6 +322,32 @@ def warnings
     redirect_to post_path(@post, params)
   end
 
+  def split
+    @page_title = 'Split Post'
+  end
+
+  def do_split
```

Comment:
Comparison - old requirements:
1\. Reply existed, was part of this post
2\. Subject given
3\. User has `split_posts` permission

New requirements (instead of 3. above):
3\. User can edit the post (either itself or metadata, same as `/posts/x/edit`)
4\. Post is author-locked

Reply:
Hm, we might want an 'or user has split_posts permission'?

Reply:
So that a mod/admin can do it? I'm not opposed if you guys think it makes sense but I don't particularly see the need.

Reply:
Yeah, I think we discussed this and decided it shouldn't be necessary with end users having access. Possible, but not necessary

Reply:
@teceler is that alright?

Reply:
Yeah, I can't think of many cases where we'd need that, unless we got called on to split a megacont post or something, and even then  you can probably just add more steps